src/rgw/rgw_d3n_cacherequest.h

```diff
@@ -135,11 +139,12 @@ struct D3nL1CacheRequest : public D3nCacheRequest {
   }
 
   int status(){
+    lsubdout(g_ceph_context, rgw_datacache, 30) << "D3nDataCache: " << __func__ << "()"<< dendl;
     lock.lock();
-    if (stat != EINPROGRESS) {
+    if (stat != EINPROGRESS){
```

Comment:
I don't think eliding space before the opening brace is preferred

Reply:
added this and several other elided spaces in this file